docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+
+  sample:
+    container_name: arc-frontend
```

Comment:
Why using a container name when you can just name the service "arc-frontend" ?

Reply:
I was following a tutorial because I'm not very familiar with setting up docker. I'll try to do the changes that you suggested.